azure-pipelines.yml

```diff
@@ -79,22 +79,22 @@ jobs:
     - script: sh -x ./clean_build.sh
       displayName: Clean Build
 
-- job: MacBuild
-  displayName: Mac Build
+- job: macOStests
+  displayName: macOS Tests
   strategy:
     matrix:
-      big-sur-Release:
+      big-sur:
           imageName: "macOS-11"
-          B_BUILD_TYPE: Release
```

Comment:
End users are likely to build with Release, so we should use that too. There are compile-time bugs that can only be seen either in Release or Debug mode.

Reply:
That has now been fixed - we have a matrix of three Mac build agents, doing two types of builds each - Debug & Release. Big Sur has a conditional during the job to generate artefacts.

Reply:
Let's save CPU time and only do release builds.

Reply:
Resolved in ecc8475e.